modules/microshift-storage-device-classes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-device-classes_{context}"]
+== Device classes
+
+You can configure device classes by adding a `device-classes` array the `lvms.yaml` configuration file. A single device class must be set as the default.
```

Comment:
@ShaunaDiaz This sentence looks a little jumbled.
Also, is the file supposed to be **lvms.yaml** or **lvmd.yaml** ?